libphal/sbe_utils.C

```diff
@@ -95,6 +95,18 @@ void mpiplContinue(struct pdbg_target *pib)
 	}
 }
 
+void mpiplEnter(struct pdbg_target *pib)
+{
+	// validate SBE state
+	validateSBEState(pib);
+
+	// call pdbg back-end function
+	auto ret = sbe_mpipl_enter(pib);
+	if (ret != 0) {
+		captureFFDC(pib);
```

Comment:
method should return ffdc data rather than relying on exceptions

Reply:
+1 it is better to return FFDC buffer

Reply:
The apis design point is converting c function to C++ way and provide addition ffdc info. that is the reason i  amde like this . anyway  made it cleaner to avoid confusion from api user. Thanks for the input